### PR TITLE
fix: fix errors and refactor

### PR DESCRIPTION
**Pull request by chandantiwari1**
fix errors and refactor code

### CODE REVIEW

**Review by abhishekrathi16 — COMMENTED** [review]
LGTM. I have fixed some merge conflicts which I encountered. The Layout not working is the only issue that needs to be resolved, resolving which the PR will be safe to merge.

**Review by abhishekrathi16 — COMMENTED** [review]
LGTM. Issues related to non-functionality of Layout still present.

**Review by abhishekrathi16 — COMMENTED** [review]
Several issues have popped up due to a minor fault of merging a branch without notice. I request that this branch be deleted and the pull request opened from the branch "merge" be merged into dev instead.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (chandantiwari1) [inline .vscode/launch.json:1]: dellete this folder

File: apps/designer/package.json [hunk 2]
```diff
@@ -1,46 +1,46 @@
 {
-    "name": "web",
+    "name": "@vvplus/web",
     "version": "0.0.0",
     "private": true,
+    "type": "module",
     "scripts": {
         "dev": "next dev",
-        "build": "prisma migrate deploy && prisma generate && next build",
```
Comment (chandantiwari1) [inline apps/designer/package.json:7]: dont change this one

File: apps/designer/package.json [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
+        "build": "next build",
         "start": "next start",
-        "lint": "next lint",
```
Comment (chandantiwari1) [inline apps/designer/package.json:9]: postinstall is necessary to generate prsma client

**Pull request closed without merging**